.github/workflows/lint-and-unit-test.yml

```diff
@@ -0,0 +1,42 @@
+name: Lint and unit test
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/**.go'
+  pull_request:
+    paths:
+      - '**/**.go'
+
+jobs:
+  lint-and-unit-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.21
+    
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo add-apt-repository universe && sudo apt-get install -y libswscale-dev libavcodec-dev libavformat-dev libavutil-dev ffmpeg
```

Comment:
We shouldn't need to install libav through apt given we are statically linking against the github repo right? Can you please remove any `apt install libav*` from all github workflows?

Reply:
Done